src/clients/googleAnalytics.ts

```diff
@@ -0,0 +1,27 @@
+type EventParameters = Parameters<(eventName: string, parameters?: Record<string, string>) => void>;
+
+const googleAnalytics = () => {
+  let gtag: Gtag.Gtag;
+  const loadGTag = (callback: (...args: EventParameters) => void) => (...args: EventParameters) => {
+    gtag = window?.gtag;
+    if (!gtag && process.env.NODE_ENV !== 'production') {
+      console.warn('gtag not loaded. Google Analytics event not sent', args);
```

Comment:
Wouldn't that crash in `production` then? As if `window.gtag` doesn't exist but the env is `production`, then this condition fails and the callback is called anyway.

Reply:
yup updating